File: src/main/java/com/faketri/market/entity/Characteristics.java
package com.faketri.market.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Entity(name = "characteristics")
@Table(schema = "public")
@AllArgsConstructor
@NoArgsConstructor
public class Characteristics {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;
    @Column
    private String name;
    @Column
    private String value;
}

File: src/main/resources/front_ts_vue/src/entities/product/model/Characteristics.ts
export default class Characteristics {
  private _id: number;
  private _name: string;
  private _value: string;

  constructor(id: number, name: string, value: string) {
    this._id = id;
    this._name = name;
    this._value = value;
  }

  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get name(): string {
    return this._name;
  }

  set name(value: string) {
    this._name = value;
  }

  get value(): string {
    return this._value;
  }

  set value(value: string) {
    this._value = value;
  }
}
